FutureNorthants/northants-design-system
# Add promo tabs component

A new component as part of the new homepage design. https://github.com/FutureNorthants/northants-website/issues/1102

This component allows 4 promo items to be in a tabbed structure to take up less space, but highlight more articles. 

Preview link available here: https://deploy-preview-533--northants-design-system.netlify.app/?path=/story/library-structure-promo-tabs--example-promo-tabs

## Testing
- Checkout this branch and run `npm install` then `npm run dev`
- Go to Structure -> Promo Tabs -> Example Promo Tabs and test styles all look ok in different browsers and that the tabs functionality works as expected. 
- Test in different screen sizes to see how it works on mobile, tablet and desktop
- Manually run tests with `npm run test`
